Sources/Jetworking/Client/Client.swift

```diff
@@ -123,6 +134,7 @@ public final class Client {
                     completion: completion
                 )
             }
+            return try taskExecutor.perform(task, on: requestExecutor)
```

Comment:
I'm not sure if this was the intention of @SimonNumberTwo that the client is throwing an error if there is no internet connection. I guess he meant that the completion of the function should be called with an error. 

@SimonNumberTwo can you give some more details about the usage in our apps? What should be the expected behavior?

Reply:
Yes exactly. I would prefer to have an error case for no internet and let the app itself using Jetworking decide what to do with it rather than implicitly throwing an error.